File: src/list/findMaxSequenceSum.ts
export default function findMaxSequenceSum(input: number[]) {
  if (input.length === 0) return 0;
  let count = 0;
  for (let index = 0; index < input.length; index++) {
    for (let idx = index; idx < input.length; idx++) {
      let total = 0;
      for (let i = index; i <= idx; i++) total += input[i];
      if (total > count) count = total;
    }
  }
  return count;
}

File: src/list/find_max_sequence_sum.py
def find_max_sequence_sum(input: list[int]) -> int:
  if len(input) == 0: return 0
  count = 0
  for index in range(0, len(input)):
    for idx in range(index, len(input)):
      local_count = 0
      for i in range(index, idx + 1):
        local_count += input[i]
      if local_count > count: 
        count = local_count
  return count
